py/README.md

```diff
@@ -83,4 +82,5 @@ poetry install
 
 ### Contribute
 
-Follow the [Sending a Pull Request](https://github.com/configu/configu/blob/main/CONTRIBUTING.md#sending-a-pull-request) section from the `CONTRIBUTING.md`.
+Follow the [Sending a Pull Request](https://github.com/configu/configu/blob/main/CONTRIBUTING.md#sending-a-pull-request)
+section from the `CONTRIBUTING.md`.
```

Comment:
why newline here?

Reply:
ide default. python project shouldn't have lines longer 79 - 119 chars. 
since it's not python, reverted to long line.